diff --git a/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts b/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts
index 79a6e83844b4..a68ccadad5bf 100644
--- a/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts
+++ b/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts
@@ -15,10 +15,6 @@ / {
 	model = "Radxa Orion O6N";
 	compatible = "radxa,orion-o6n";
 
-	chosen {
-		stdout-path = &uart2;
-	};
-
 	dummy_codec: dummy_codec {
 		#sound-dai-cells = <1>;
 		compatible = "linux,dummy";
@@ -659,13 +655,13 @@ CIX_PAD_GPIO136_FUNC_GPIO136 (PULL_DOWN|ST|DS_LEVEL12) /* GPIO136 for touch pane
 
 	pinctrl_fch_pwm0: pinctrl_fch_pwm0 {
 		sky1,pins = <
-			CIX_PAD_DP2_VARY_BL_FUNC_DP2_VARY_BL (DS_LEVEL7)
+			CIX_PAD_UART0_TXD_FUNC_PWM0 (PULL_DOWN|ST|DS_LEVEL7)
 		>;
 	};
 
 	pinctrl_fch_pwm1: pinctrl_fch_pwm1 {
 		sky1,pins = <
-			CIX_PAD_UART1_TXD_FUNC_FAN_OUT0 (PULL_DOWN|ST|DS_LEVEL7)
+			CIX_PAD_UART0_RXD_FUNC_PWM1 (PULL_DOWN|ST|DS_LEVEL7)
 		>;
 	};
 
@@ -990,6 +986,7 @@ CIX_PAD_I2S2_DATA_IN0_FUNC_GPIO075 (PULL_UP|DS_LEVEL4) /* GPIO0075 */
 	};
 };
 
+#if 0
 &pwm0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_fch_pwm0>;
@@ -1001,6 +998,7 @@ &pwm1 {
 	pinctrl-0 = <&pinctrl_fch_pwm1>;
 	status = "okay";
 };
+#endif
 
 &hifi5 {
 	status = "okay";
