diff --git a/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts b/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts
index 79a6e83844b4..f1991e0d4141 100644
--- a/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts
+++ b/arch/arm64/boot/dts/cix/sky1-orion-o6n.dts
@@ -15,10 +15,6 @@ / {
 	model = "Radxa Orion O6N";
 	compatible = "radxa,orion-o6n";
 
-	chosen {
-		stdout-path = &uart2;
-	};
-
 	dummy_codec: dummy_codec {
 		#sound-dai-cells = <1>;
 		compatible = "linux,dummy";
@@ -2194,7 +2190,7 @@ &its_pcie {
 };
 
 &smmu_pciehub {
-	status = "disabled";
+	status = "okay";
 };
 
 &smmu_mmhub {
